packages/plugins/slugManager/manager.ts

```diff
@@ -54,7 +54,7 @@ export function createManager(config: InternalSlugManagerConfig) {
     const entitiesResponse = await api.listEntities(params);
 
     const profileParams = new URLSearchParams({
-      fields: ["meta", slugField, ...(slugGeneratorFnFields || [])].join(","),
+      fields: ["meta", slugField, ...slugGeneratorFnFields].join(","),
```

Comment:
It was already there, but wondering if there's a reason we always request `slugField` here?

Reply:
I think it was added because `slug` is included as an optional value for the `BaseEntity` interface, but I think we should remove since its possible that slugField isn't necessarily always equal to "slug".